Apps/WebClient/src/ClientApp/src/components/private/home/HomeView.vue

```diff
@@ -474,6 +499,35 @@ watch(vaccineRecordState, () => {
                 </p>
             </HgCardComponent>
         </v-col>
+        <v-col
+            v-if="showRecommendationsCardButton"
+            :cols="getGridCols"
+            class="d-flex"
+        >
+            <HgCardComponent
+                data-testid="recommendations-card-btn"
+                class="flex-grow-1"
+                @click="showRecommendationsDialog()"
+            >
+                <template #icon>
+                    <img
+                        class="quick-link-icon-large"
+                        src="@/assets/images/services/immunize-bc-logo.svg"
+                        alt="Immunize BC Logo"
+                    />
+                </template>
+                <template #menu-items>
+                    <v-list-item
+                        data-testid="remove-quick-link-button"
+                        title="Remove"
+                        @click.stop="handleClickRemoveRecommendations()"
+                    />
+                </template>
+                <p class="text-body-1">
+                    View recommendations for vaccines your are eligible for.
```

Comment:
```suggestion
                    View recommendations for vaccines you are eligible for.
```
Typo in acceptance criteria.